src/pages/AdminTraineeDashboard.tsx

```diff
@@ -0,0 +1,266 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import Sidebar from '../components/Siderbar';
+
+const AdminTraineeDashboard = () => {
+  const [registerTraineeModel, setRegisterTraineeModel] = useState(false);
+
+  const removeModel = () => {
+    let newState = !registerTraineeModel;
+    setRegisterTraineeModel(newState);
+  };
+
+  let developers = [
```

Comment:
move these data into a file and import them from there.

We will replace these after adding a json server

Reply:
solved